kubernetes/assets/recommended_monitors/monitor_node_unavailable.json

```diff
@@ -0,0 +1,30 @@
+{
+	"name": "[kubernetes] Monitor Unavailable Kubernetes Nodes",
+	"type": "query alert",
+	"query": "max(last_15m):sum:kubernetes_state.node.status{status:schedulable} by {kubernetes_cluster} * 100 / sum:kubernetes_state.node.status{*} by {kubernetes_cluster} < 80",
+	"message": "Nodes are unavailable on ({{kubernetes_cluster.name}} cluster)",
```

Comment:
have a better message here - e.g. more than 20% of the nodes are unschedulable (not unavailable).

Reply:
mention that this might be expected